content/doc/book/managing/plugins.adoc

```diff
@@ -335,3 +335,46 @@ created/deleted to control the pinning behavior. If the `pinned` file is
+When a plugin was attached to the Jenkins core before being detached, it may or may not have used any of its functionality in other plugins that relied on that version of Jenkins.
+To guarantee that plugins do not fail when the functionality on which they rely is separated from Jenkins core, it is regarded to have a dependency on the detached plugin if it specifies a dependency on a version of Jenkins core prior to the split.
+It is assumed that there is a dependency on the detached plugin even though it isn't stated explicitly.
+
+Implied dependencies build up over time for plugins that don't frequently update which Jenkins core version they depend on.
+
+=== Why does the implied dependency exist?
+
+Because the WMI Windows Agent plugin was separated from Jenkins core, Jenkins core is uninformed if the dependent plugin needs any functionality that was previously present in Jenkins core.
+This creates an implied dependency.
+
+It is "implied" since it is not stated explicitly in the plugin, and it is a dependency because Jenkins core is uncertain if the APIs available in Jenkins core prior to the separation of the plugin are necessary or not.
+
+[NOTE]
+====
+Release 1.547 of Jenkins separated the WMI Windows Agent plugin from the main code.
+Jenkins core assumes a dependency on plugins that made a declaration of dependence on a previous version of Jenkins core.
+====
+
+The Jenkins administrator can see the plugins that have an implied dependency from the plugin manager page. 
+They hover over the "uninstall" button and a list of plugins with an implied dependency is displayed.
+
+
+=== How can the implied dependency be removed?
+
+The implied dependency can be removed by releasing a new version of the plugin that depends on a newer minimum Jenkins core than Jenkins 1.547. 
+The link:https://www.jenkins.io/doc/developer/tutorial-improve/[Improve a plugin tutorial] provides steps that can assist plugin developers as they configure the plugin to link:https://www.jenkins.io/doc/developer/tutorial-improve/update-base-jenkins-version/[depend on a newer minimum Jenkins core].
+
+
+=== Which plugins have an implied dependency on WMI Windows Agents plugin?
+
+As of 16 Dec 2022, there were over 300 Jenkins plugins that declare a minimum Jenkins version before 1.547.
+Those plugins are listed in the link:https://docs.google.com/spreadsheets/d/1KDIJIu41rIRlj6PjzyhAY9vazKmd3PVwqM3_-32Lt-o/edit?usp=sharing["Plugins with implied dependency on WMI Windows Agents" worksheet].
+
+link:https://issues.jenkins.io/secure/Dashboard.jspa[Jira issues] for those plugins to upgrade their minimum Jenkins version can be linked in this epic as a convenient way of seeing the related issues.
+A Jira issue is not required in order to release a new version of an affected plugin.
+The Jira issue is just a convenience in case it helps someone justify that new release or explain why it is not being released.
```

Comment:
```suggestion
=== Which plugins have an implied dependency on WMI Windows Agents plugin?

As of December 16 2022, there were over 300 Jenkins plugins that declare a minimum Jenkins version before 1.547.
Those plugins are listed in the link:https://docs.google.com/spreadsheets/d/1KDIJIu41rIRlj6PjzyhAY9vazKmd3PVwqM3_-32Lt-o/edit?usp=sharing["Plugins with implied dependency on WMI Windows Agents" worksheet].

GitHub issues for plugins to upgrade their minimum Jenkins version can be linked in link:https://issues.jenkins.io/browse/JENKINS-70301[this epic] as a convenient way of seeing the related issues.
A Jira issue is not required in order to release a new version of an affected plugin.
The Jira epic provides a unified location for tracking and can help someone justify the new release or explain why it is not being released.
```
updated the syntax and content here, there were a couple of updates needed for the proper linking/connecting to the epic, and the plugins are more likely than not tracking issues in GitHub, as all the PRs created in the google sheet lead to the respective plugin repository. The epic provides a single place to link to these PRs for convenience.